liri.js

```diff
@@ -0,0 +1,136 @@
+var keys = require("./keys.js");
+var request = require("request");
+var Twitter = require("twitter");
+var twitter = new Twitter(keys.twitKeys);
+var Spotify = require('node-spotify-api');
+var spot = new Spotify(keys.spotKeys);
+var fs = require("fs");
+var command = process.argv[2];
+var input = process.argv.slice(3);
+
+if (input.length !== 0) input = addSpaces(input);
+
+
+if (input.length !== 0) {
+  doSomething(command, input);
+} else {
+  doSomething(command);
+}
+
+function doSomething(task, argument = "") {
+  write(task + " " + argument);
+  switch (task) {
+    case "my-tweets":
+      if (argument !== "") {
+        myTweets(argument);
+      } else {
+        myTweets();
+      }
+      break;
+    case "spotify-this-song":
+      if (argument !== "") {
+        spotifyThis(argument);
+      } else {
+        spotifyThis();
+      }
+      break;
+    case "movie-this":
+      if (argument !== "") {
+        movieThis(argument);
+      } else {
+        movieThis();
+      }
+      break;
+    case "do-what-it-says":
+    case "trust-me":
+      trustMe();
+      break;
+    default:
+  }
+}
+
+function addSpaces(words) {
+  if (words[0].indexOf(" ") >= 0) {
+    words = words[0].split(" ");
+  }
+  var text = words.join(" ");
+  return text;
+}
+
+//Doesn't use my twitter. I barely tweet. 
+//So here is a way better twitter user Jeff Gerstmann.
+//Or you can enter any screen name to get that user.
+function myTweets(name = 'jeffgerstmann') {
+  var params = { screen_name: name };
+  twitter.get('statuses/user_timeline', params, function(error, tweets, response) {
+    if (!error && response.statusCode === 200) {
```

Comment:
This can just be:
```javascript
 if (response.statusCode === 200) {
```
if it erros this wouldn't fire anyways, besides, it is better to check for errors not check if there was no errors.